Testing/PythonTests/crop.py

```diff
@@ -17,11 +17,9 @@ def cropTest1():
   return img.compare(compareImg)
 
 try:
-  if not cropTest1():
```

Comment:
This is actually correct, and the modification is incorrect. An exception is indeed a failure, but the test itself needs to verify the expected result.